packages/lesswrong/components/messaging/FriendlyInbox.tsx

```diff
@@ -51,14 +51,22 @@ const styles = (theme: ThemeType): JssStyles => ({
     minHeight: 0,
     display: "flex",
     flexDirection: "row",
-    flex: 1
+    flex: 1,
+    overflow: "hidden", // to simplify border radius
+    border: theme.palette.border.grey200,
+    borderRadius: `${theme.borderRadius.default}px ${theme.borderRadius.default}px 0px 0px`,
+    backgroundColor: theme.palette.background.pageActiveAreaBackground,
```

Comment:
The way I had done the borders before was less than ideal, I've simplified it here (mainly moved the outer border to the `table` class)